fe/fe-core/src/main/java/com/starrocks/catalog/Type.java

```diff
@@ -147,6 +152,25 @@
                     .add(JSON)
                     .build();
 
+    protected static final ImmutableList<Type> SUPPORT_SCALAR_TYPE_LIST =
+            ImmutableList.copyOf(SUPPORTED_TYPES.stream().filter(Type::isScalarType).collect(Collectors.toList()));
+
+    protected static final ImmutableSortedMap<String, ScalarType> STATIC_TYPE_MAP =
+            ImmutableSortedMap.<String, ScalarType>orderedBy(String.CASE_INSENSITIVE_ORDER)
+                    .put("DECIMAL", Type.DEFAULT_DECIMALV2) // generic name for decimal
+                    .put("STRING", Type.DEFAULT_STRING)
+                    .put("INTEGER", Type.INT)
```

Comment:
Why add "INTEGER" alone, it looks like can not create Type.INT by this name before

Reply:
`INTEGER` is an alias of `INT`. There's some test cases cover it.

Reply:
The parser could accept this token, in `AstBuilder.java:1548`.